ci-operator/jobs/openshift/ovn-kubernetes/openshift-ovn-kubernetes-master-presubmits.yaml

```diff
@@ -10,7 +10,7 @@ presubmits:
       skip_cloning: true
     labels:
       ci-operator.openshift.io/prowgen-controlled: "true"
-    name: pull-ci-openshift-ovn-kubernetes-master-e2e-aws
+    name: pull-ci-openshift-ovn-kubernetes-master-e2e-aws-openshift-sdn
```

Comment:
We should remove the openshift-sdn-based ovn-kubernetes test from the openshift/ovn-kubernetes repo entirely. There is no point in running it from there.